roles/cs.nginx-https-termination/defaults/main.yml

```diff
@@ -78,3 +78,12 @@ https_termination_hosts:
 # Internal helper var for storing runtime computed configuration
 https_termination_cfg_vhosts: []
 
+# Rewrite JS cookies into server-created (first party) cookies to work around
+# short expiration time forced on JS cookies by some browsers (Safari, ...).
+# Shall be a mapping between JS cookie name -> HTTP cookie name.
+https_termination_nginx_server_cookie_rewrite_map:
+    # DynamicYield tracking cookie
+    _dyid: _dyid_server
```

Comment:
I would move it to project settings. Very project specific

Reply:
You're right, removed defaults, but left this config in comment as an example #4295a0d